PHP-Variables.md

```diff
@@ -0,0 +1,162 @@
+# PHP : Variables
+
+Variables are containers for storing data such as Strings, Integers, Boolean values, Arrays and Objects.
```

Comment:
> for storing data such as Strings, Integers, Boolean values, Arrays and Objects.

Are those keywords?

If so, it would be

> for storing data such as `Strings`, `Integers`, `Boolean` values, `Array`s and `Object`s.